hub/api/tests/test_api.py

```diff
@@ -195,7 +195,7 @@ def test_empty_samples(ds: Dataset):
 
     # test indexing individual empty samples with numpy while looping, this may seem redundant but this was failing before
     for actual_sample, expected in zip(ds, expected_list):
-        actual = actual_sample.with_empty.numpy()
+        actual = actual_sample["with_empty"].numpy()
```

Comment:
wait, did we remove `.` syntax for accessing tensors in datasets? if not, can you change back to `actual_sample.with_empty`

Reply:
Mypy complains.